ansible/mediawiki-install.yml

```diff
@@ -0,0 +1,228 @@
+# Notes before starting:
+#
+# Set up /etc/ansible/hosts to include any hosts for which you want to
+# run this playbook, within a `[mediawiki]` group.  Test that you can
+# ping these hosts with ansible and the desired user.  If they are not
+# already in known_hosts, you may get an error.  Either add them to
+# known_hosts or edit the configuration to ignore that check.  See
+# https://docs.ansible.com/ansible/latest/user_guide/intro_getting_started.html
+# for general help with Ansible.
+
+---
+- hosts: default ansible_host=127.0.0.1 ansible_port=2222 ansible_user='vagrant' ansible_ssh_private_key_file='~/.ssh/vagrant-insecure'
+  remote_user: root
+  tasks:
+
+  - name: install prerequisites
+    # See https://www.mediawiki.org/wiki/Manual:Installation_requirements
+    apt:
+      name: "{{ item }}"
+      update_cache: yes
+      state: present
+    loop:
+      - apache2
+      - curl
+      - default-mysql-server
+      - default-mysql-client
+      - git
+      - php
+      - php-apcu
+      - php-cli
+      - php-intl
+      - php7.3-mbstring
```

Comment:
I had to add a 'php7.2-mbstring' here.  For some reason the php 7.3 version wasn't being enabled.